types/@rebass/grid.d.ts

```diff
@@ -0,0 +1,241 @@
+export namespace Box {
```

Comment:
Did you copy explicitly this typings?

Reply:
I should have executed command into terminal to use rebass/grid, and this installed me folder 'types'

Reply:
You can use it from node_modules so please remove it